backend/src/main/resources/reference.conf

```diff
@@ -3,14 +3,16 @@ bootzooka {
 
   db {
     h2 {
+      queueSize = -1
       dataSourceClass = "org.h2.jdbcx.JdbcDataSource"
       properties = {
         url = "jdbc:h2:file:./data/bootzooka"
       }
     }
     postgres {
+      queueSize = -1
       dataSourceClass = "org.postgresql.ds.PGSimpleDataSource"
-      maxConnections = 16
+      maxConnections = 20
```

Comment:
I should probably comment this in the code, but I found 16 to be a good value on the default (free) postgres-heroku database. The connection limit there is 20, minus a couple of connections from GUI/command line clients, hence 16